Jenkinsfile_servlet

```diff
@@ -28,44 +27,81 @@ node('linux'){
               globalMavenSettingsConfig: 'oss-settings.xml',
               mavenOpts: '-Xms2g -Xmx4g',
               mavenLocalRepo: ".repository") {
-        sh "mvn -V -B clean install -DskipTests -T10 -e"
+        sh "mvn -V -B -pl jetty-home -am clean install -DskipTests -T2 -e"
       }
     }
   }
   stage("Setup servlet tck"){
-    env.JAVA_HOME='${tool "$jdk"}'
-    env.PATH="${env.JAVA_HOME}/bin:${env.PATH}"
-    sh "wget https://jenkins.webtide.net/userContent/servlettck-4.0_latest.zip"
-    sh "unzip servlettck-4.0_latest.zip"
+    echo "Starting withEnv()"
+    withEnv(["JAVA_HOME=${ tool "$jdk" }", "PATH=${ tool "$jdk" }/bin:${env.PATH}"]) {
+      // env.JAVA_HOME='${tool "$jdk"}'
+      // env.PATH="${env.JAVA_HOME}/bin:${env.PATH}"
+
+      echo "JAVA_HOME=${env.JAVA_HOME}"
+      echo "PATH=${env.PATH}"
+
+      sh "ls -la ${env.JAVA_HOME}"
+
+      // Execute some simple lookups of required command line applications
+      // If the application isn't found, then it fails the build
+      sh "which curl"
+      sh "which wget"
+      sh "which unzip"
+      sh "which find"
+      sh "which java"
```

Comment:
This block was useful, as it quickly told me if I had a missing command line application.